cmds/tui.c

```diff
@@ -2002,15 +1994,17 @@ static bool tui_window_collapse(struct tui_window *win)
 	if (win->ops->collapse == NULL)
 		return false;
 
-	return win->ops->collapse(win, win->curr);
+	/* fold all the directly children */
+	return win->ops->collapse(win, win->curr, 1);
```

Comment:
This changes the current behavior - do you think it's worth changing when it comes to usability?

Reply:
It changes to unfold the current node first then folds all its children. I think it's very much needed because previous behavior doesn't make any visible change when the current node is already folded.